egs/csj/s5/local/rnnlm/run_rnnlm.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+# Begin configuration section.
+# This recipe is based on wsj/s5/local/rnnlm/run_rnnlm.sh
+
+# without RNNLM
+# eval1: %WER 10.05 [ 2615 / 26028, 274 ins, 494 del, 1847 sub ] exp/chain/tdnn1b/decode_eval1/wer_10_0.5
+# eval2: %WER 8.09 [ 2157 / 26661, 177 ins, 394 del, 1586 sub ] exp/chain/tdnn1b/decode_eval2/wer_12_0.5
+# eval3: %WER 9.33 [ 1603 / 17189, 169 ins, 304 del, 1130 sub ] exp/chain/tdnn1b/decode_eval3/wer_13_0.5
+
+# with RNNLM
+# eval1: %WER 8.52 [ 2218 / 26028, 239 ins, 454 del, 1525 sub ] exp/chain/tdnn1b/decode_eval1_lstm_rnnlm/wer_12_0.0
+# eval2: %WER 7.37 [ 1964 / 26661, 227 ins, 259 del, 1478 sub ] exp/chain/tdnn1b/decode_eval2_lstm_rnnlm/wer_10_0.0
+# eval3: %WER 7.83 [ 1346 / 17189, 140 ins, 260 del, 946 sub ] exp/chain/tdnn1b/decode_eval3_lstm_rnnlm/wer_13_0.5
+
+dir=exp/rnnlm_tdnn1b
+embedding_dim=800
+lstm_rpd=200
+lstm_nrpd=200
+embedding_l2=0.001 # embedding layer l2 regularize
+comp_l2=0.001 # component-level l2 regularize
+output_l2=0.001 # output-layer l2 regularize
+epochs=20
+stage=0
+train_stage=-10
+
+# variables for rnnlm rescoring
+ac_model_dir=exp/chain/tdnn1b
+ngram_order=4
+decode_dir_suffix=lstm_rnnlm
+
+. ./cmd.sh
+. ./utils/parse_options.sh
+[ -z "$cmd" ] && cmd=$train_cmd
+
+export PYTHONIOENCODING=UTF-8
```

Comment:
avoid UnicodeDecodeError.

Reply:
Where does this error happen?

Reply:
I had got an error like `UnicodeEncodeError: Ascii codec can't encode character *** in position 0:` without inserting a line `export PYTHONIOENCODING=UTF-8` before.
Although I tried to reproduce the error to find out where it occurred, I couldn't get it. 
No error was happened without this line.
It may be because I changed my environment since then.

I erase this line.

Reply:
OK, if it happens again we'll try to track it down.